spec/factories/projects.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :project do
+    name "MyText"
```

Comment:
Can we use Faker here to that we know we're differentiating between generated projects?